tests/unit/modules/boto_iot_test.py

```diff
@@ -98,6 +98,7 @@ def _has_required_boto():
                       ruleDisabled=True)
 
 
+@skipIf(True, 'These tests are pegging the CPU and eating all available RAM')
 class BotoIoTTestCaseBase(TestCase):
```

Comment:
@kraney would you also be able to look at the other test failures that are here? It looks like when the initial PR was made, we didn't have boto3 being put on the test boxes, so these tests never actually ran.